test/cmdlineTests.sh

```diff
@@ -133,13 +135,9 @@ function test_solc_behaviour()
 
     if [[ "$exit_code_expected" = "" ]]; then exit_code_expected="0"; fi
 
-    local solc_command="$SOLC ${filename} ${solc_args}"
-    if [[ -n "$solc_stdin"  ]]; then solc_command+=" <$solc_stdin"  ; fi
-    if [[ -n "$stdout_path" ]]; then solc_command+=" 1>$stdout_path"; fi
-    if [[ -n "$stderr_path" ]]; then solc_command+=" 2>$stderr_path"; fi
-
+    local solc_command="$SOLC ${filename} ${solc_args} <$solc_stdin"
     set +e
-    eval "$solc_command"
+    "$SOLC" "${filename}" ${solc_args} <"$solc_stdin" >"$stdout_path" 2>"$stderr_path"
```

Comment:
Shouldn't this be

`"$SOLC" "${filename}" ${solc_args} <"$solc_stdin" 1>"$stdout_path" 2>"$stderr_path"`

Reply:
You mean adding the `1`? Isn't it equivalent?

Reply:
Yes, this is optional (for clarity)